generic-database-plugin/widgets/Database-action.json

```diff
@@ -9,20 +9,12 @@
       "properties": [
         {
           "widget-type": "textbox",
-          "label": "Plugin Name",
-          "name": "jdbcPluginName"
+          "label": "Driver Name",
+          "name": "jdbcDriverName"
```

Comment:
Changing the label from "Plugin Name" to "Driver Name" is good. But let's not change the name in the backend because that breaks backwards compatibility. Older pipelines that are upgraded will no longer work.

Reply:
fixed